bundles/org.openhab.binding.androidtv/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -43,9 +43,15 @@
 				<label>Hostname</label>
 				<description>Hostname or IP address of the device</description>
 			</parameter>
-			<parameter name="port" type="integer">
-				<label>Port</label>
+			<parameter name="googletvPort" type="integer">
+				<label>GoogleTV Port</label>
 				<description>Port to connect to</description>
+				<default>6466</default>
+			</parameter>
+			<parameter name="shieldtvPort" type="integer">
```

Comment:
Can this parameter be marked as advanced if the default port will work for most users?